scripts/clan/baseclan.py

```diff
@@ -0,0 +1,42 @@
+
+class BaseClan:
+    """The base clan in ClanGen, which living Clans derive from."""
+
+    name = ""
+
+    BIOME_TYPES = ["Forest", "Plains", "Mountainous", "Beach"]
+    SEASONS = [
+        "Newleaf",
+        "Newleaf",
+        "Newleaf",
+        "Greenleaf",
+        "Greenleaf",
+        "Greenleaf",
+        "Leaf-fall",
+        "Leaf-fall",
+        "Leaf-fall",
+        "Leaf-bare",
+        "Leaf-bare",
+        "Leaf-bare",
+    ]
+
+    current_season = "Newleaf"
+    starting_season = "Newleaf"
+    biome = "Plains"
```

Comment:
more a request than an issue, but can the default biome be forest? it was the First and Only biome for a while, so it makes more sense in my brain :3

Reply:
Fixed :) That does make more sense to me too haha